src/cpu/x64/jit_brgemm_conv_utils.cpp

```diff
@@ -423,7 +423,7 @@ struct brg_blocking_t : public jit_brgemm_conv_conf_t {
     static constexpr int bcast_simd = 16;
 
     int sp, sp_block, nb_sp;
-    static int last_ic_block_size;
+    static thread_local int last_ic_block_size;
```

Comment:
As far as I understand we could just use
`int last_ic_block_size`
instead.
Do we expect to have real benefits by using 'static thread_local' instead?

Reply:
@EgorDuplensky  Yes, removing static also is good, but we need to update many codes, because "last_ic_block_size" is used for many funcitons. If we only add thread_local for this static variable, we just have 2 changes. I think only add thread_local is more concise.
BTW, I also filed a jira for oneDNN team. https://jira.devtools.intel.com/browse/MFDNN-9061

Reply:
oneDNN team mentioned, that they already prepared the fix as well.
I propose to wait a bit and see if it is the same.
Just to make sure we do not bring additional diff to the fork.
@dmitry-gorokhov FYI